MAUI/DataGrid/Editing.md

```diff
@@ -0,0 +1,376 @@
+# Editing in MAUI DataGrid (SfDataGrid)
+
+The [Syncfusion .NET MAUI DataGrid]() (SfDataGrid) control supports editing cell values by setting the [SfDataGrid.AllowEditing property](), [SfDataGrid.NavigationMode]() to Cell, and setting the [SfDataGrid.SelectionMode]() to any value other than None.
+
+To enable editing, follow the code example:
+
+{% tabs %}
+{% highlight xaml %}
+    <syncfusion:SfDataGrid x:Name="dataGrid"
+                           AllowEditing="True"
+                           SelectionMode="Multiple"    
+                           NavigationMode="Cell" 
+                           ItemsSource="{Binding Orders}" />
+{% endhighlight %}
+{% highlight c# %}
+dataGrid.AllowEditing = true;
+dataGrid.SelectionMode = SelectionMode.Multiple;
+dataGrid.NavigationMode = NavigationMode.Cell;
+{% endhighlight %}
+{% endtabs %}
+
+![DataGrid with editing](Images\editing\maui-datagrid-editing.png)
+
+## Column editing
+
+To enable or disable editing for a specific column, you need to set the [DataGridColumn.AllowEditing]() property.
+
+{% tabs %}
+{% highlight xaml %}
+    <syncfusion:DataGridTextColumn AllowEditing="True"
+                                   MappingName="OrderID" />
+{% endhighlight %}
+{% highlight c# %}
+DataGridTextColumn column = new DataGridTextColumn();
+column.MappingName="OrderID";
+column.AllowEditing = false;
+{% endhighlight %}
+{% endtabs %}
+
+N>The `DataGridColumn.AllowEditing` takes higher priority than the `SfDataGrid.AllowEditing`.
+
+## Entering into edit mode
+
+To enter edit mode by simply tapping or double-tapping the grid cells, set the [SfDataGrid.EditTapAction]() property.
```

Comment:
To enter the edit mode, you can simply tap or double-tap the grid cell. The behavior of the edit mode tapping is controlled by the SfDataGrid.EditTapAction property, with the default value being EditTapAction.DoubleTap.

Reply:
![image](https://github.com/syncfusion-content/maui-docs/assets/113881396/6c27592c-61b0-440e-ba92-e0a155c1cc82)